### DISCUSSION TITLE
Set the default ref to main

### DISCUSSION THREAD

**Discussion by white-gecko** [gollum/gollum#1833] [General]
Since the default branch often is main (https://github.com/gollum/gollum/issues/1813) I can specify it when running with the `--ref` option. Is there a possibility to specify it in some default configuration file?

**Comment by oeyoews** [comment]
edit config.rb file,  use `ref: "main"` option, @white-gecko

**Comment by white-gecko** [comment]
How do I add this? If I just write
```
ref: "main"
```

into `conifg.rb` it still looks for `master`.

**Reply by oeyoews** [reply]
use this cmd `gollum --port 8082 --config config.rb` launch gollum @white-gecko 

[ref](https://gitlab.com/oeyoews/md-in-web/tree/main)

**Reply by dometto** [reply]
The `--port` component is superfluous. Just use `--config config.rb`, or just `--config`, and look at the available options here:

https://github.com/gollum/gollum/wiki/Sample-config.rb

**Reply by oeyoews** [reply]
![image](https://user-images.githubusercontent.com/72405338/183604393-44ba4330-1a8a-4f22-953c-cccdc7e8d1d0.png)
- this option maybe not work
- [ref](https://github.com/gollum/gollum/discussions/1838#discussioncomment-3345897)

**Reply by dometto** [reply]
This is right, but OP was not asking about the `--port` flag.

**Comment by dometto** [comment]
For future reference, what you need to set in your config file is:

```ruby
wiki_options = {
  ref: 'main'
}
```
